vendor/nouislider/shim.js

```diff
@@ -0,0 +1,9 @@
+/* global define, noUiSlider */
+
+define('noUiSlider', [], function() {
+  "use strict";
+
+  return {
+    'default': noUiSlider
+  };
+});
```

Comment:
This shim allows us to [import](https://github.com/kennethkalmer/ember-cli-nouislider/pull/10/files#diff-e7826431cfe918a362498d5d123d8effR2) the `noUiSlider` global package in the component.